wrappers/kn/README.md

```diff
@@ -0,0 +1,56 @@
+# ZXing-C++ Kotlin/Native Library
+
+## Install
+
+The easiest way to use the library is to fetch if from _mavenCentral_. Simply add
+
+```gradle
+implementation("io.github.zxing-cpp:kotlin-native:2.2.1")
+```
+
+to your `build.gradle.kts` file in the `dependencies` section of `nativeMain` source set.
+
+## Use
+
+A trivial use case looks like this:
+
+```kotlin
+import zxingcpp.BarcodeFormat
+import zxingcpp.BarcodeReader
+import zxingcpp.ImageFormat
+import zxingcpp.ImageView
+
+val data: ByteArray = ...    // the image data
+val width: Int = ...         // the image width
+val height: Int = ...        // the image height
+val format = ImageFormat.Lum // ImageFormat.Lum assumes grey scale image data
+
+val image = ImageView(data, width, height, format)
+val barcodeReader = BarcodeReader().apply {
+   formats = setOf(BarcodeFormat.EAN13, BarcodeFormat.QRCode)
+   tryHarder = true
+   maxNumberOfSymbols = 3
+   // more options, see documentation
+}
+
+barcodeReader.read(image).joinToString("\n") { barcode: Barcode ->
+   "${barcode.format} (${barcode.contentType}): ${barcode.text}"
+}
+```
+
+Here you have to load your image into memory by yourself and pass the decoded data to the constructor of `ImageView`.
+
+## Build locally
+
+1. Install JDK, CMake and Android NDK(With `$ANDROID_NDK` correctly configured) and ensure their
+   executable binaries appear in `$PATH`.
+2. Prepare kotlin/native toolchain (You can easily do this by cloning
+   [K/N Toolchain Initializer](https://github.com/ISNing/kn-toolchain-initializer) and executing `gradle build`
+   so that kotlin will download toolchains needed into user's home dir.).
+3. Ensure there's `run_konan` available in `$PATH` or specify path of kotlin-native toolchain in `local.properties`
+   like `konan.dir=/home/user/.konan/kotlin-native-prebuilt-linux-x86_64-1.9.22`.
+5. Ensure there's `llvm-ar` available in `$PATH` for mingwX64 target building.
```

Comment:
Obviously just polish but since you already put so much work into this, why not make it perfect? This line should start with a '4' not a '5' ;)

Reply:
fixed in https://github.com/zxing-cpp/zxing-cpp/pull/719/commits/41944322626cb42a52b61ad2416e785d9471dd23